ginga/web/pgw/Widgets.py

```diff
@@ -1728,11 +2113,75 @@ def render(self):
         return self.html_template % d
 
 
-class StackWidget(TabWidget):
+class StackWidget(ContainerBase):
+
+    html_template = """
+    <div id='%(id)s' class="%(classes)s" style="%(styles)s">
+        %(content)s
+    </div>
+    """
+
     def __init__(self):
-        super(StackWidget, self).__init__(tabpos='top', reorderable=False,
-                                          detachable=False, group=-1)
-        self._tabs_visible = False
+        super(StackWidget, self).__init__()
```

Comment:
I think you can use `super()` now without filling in the stuff. But it is also okay if you want to wait to remove remnant Python 2 styles in a separate PR.

```suggestion
        super().__init__()
```

Reply:
> But it is also okay if you want to wait to remove remnant Python 2 styles in a separate PR.

That makes sense to me.  Let's do that.